src/components/search/Search.jsx

```diff
@@ -1,9 +1,39 @@
-import React from 'react'
+import React, { Component } from "react";
+import "./_Search.scss";
+import searchIcon from "assets/search-solid.png";
```

Comment:
Now that font awesome has been added, you can use it instead of an image

Reply:
Attended